pkg/snapshot/store.go

```diff
@@ -277,7 +286,35 @@ func ConfigureStore(ctx context.Context, options ConfigureStoreOptions) (*types.
 			return nil, errors.Wrap(err, "failed to get k8s clientset")
 		}
 
-		storeFileSystem, err := BuildStoreFileSystem(ctx, clientset, options.KotsadmNamespace)
+		storeFileSystem, err := BuildMinioStoreFileSystem(ctx, clientset, options.KotsadmNamespace)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to build file system store")
+		}
+		store.FileSystem = storeFileSystem
+	} else if options.FileSystem != nil && options.IsMinioDisabled {
+		store.AWS = nil
+		store.Google = nil
+		store.Azure = nil
+		store.Other = nil
+		store.Internal = nil
+
+		b, err := json.Marshal(options.FileSystem)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to marshal filesystem config")
+		}
+
+		hash := md5.Sum(b)
+		hashId := hex.EncodeToString(hash[:6])
+		store.Bucket = "velero-lvp-" + hashId
```

Comment:
why not use a fixed bucket name? won't this start using a different bucket if we add more options to the file system config later?

Reply:
Mostly just forward looking that maybe we will support multiple backup storage locations someday, and this would prevent naming collisions. The bucket name is used as the name of the volume attached to Velero/Restic. LVP plugin should replace the volume mount name if the path changes, so I can stick with a fixed name if you feel strongly about it.